### ISSUE TITLE
invalid format for MountPoint in WSL2

### ISSUE THREAD

**Issue by rue-nsilverman** [uber-go/automaxprocs#65]
The most recent version of [goreleaser](https://github.com/goreleaser/goreleaser/pull/3901) includes this module and I have encountered an error running goreleaser in wsl. I use Docker Desktop using the WSL engine which creates local mounts inside linux to windows paths. The exact error is:
```
invalid format for MountPoint: "2619 80 0:302 / /Docker/host rw,noatime - 9p drvfs rw,dirsync,aname=drvfs;path=C:\\Program Files\\Docker\\Docker\\resources;symlinkroot=/mnt/,mmap,access=client,msize=262144,trans=virtio"
```
I assume given the a quick scan over the code, that the way the mount line is split is breaking because of the weird spaces and double backspaces in the path option. The version I am using is 1.5.2 of this module.

**Comment by hhromic** [comment]
I just encountered this exact same issue while using the latest version of goreleaser as well :(

**Comment by hhromic** [comment]
I went to investigate a bit more and the root of the problem is Docker Desktop for Windows creating an invalid mountpoint in `/proc/X/mountinfo`. Reference: https://github.com/docker/for-win/issues/13318

Automaxprocs is not the only software failing to parse the spaces in the `path` subfield in the options field.

The error only happens when Docker Desktop is integrated into the WSL distribution and _running_. If Docker Desktop is stopped, the mountpoint is unmounted and automaxprocs (and other parsing software) does not error anymore.

In other words, automaxprocs is actually parsing the `mountinfo` metafile correctly (as defined in the official [kernel documentation](kernel.org/doc/Documentation/filesystems/proc.txt)) and is Docker Desktop who should fix this issue.

**Comment by sywhang** [comment]
Should be fixed in [v1.5.3 release](https://github.com/uber-go/automaxprocs/releases/tag/v1.5.3)
